pkg/api/install/install.go

```diff
@@ -13,6 +13,7 @@ import (
 	_ "github.com/openshift/origin/pkg/oauth/api/install"
 	_ "github.com/openshift/origin/pkg/project/api/install"
 	_ "github.com/openshift/origin/pkg/route/api/install"
+	_ "github.com/openshift/origin/pkg/scheduler/admission/podnodeconstraints/api/install"
```

Comment:
these moved from the main api install to the server config api install file:
https://github.com/openshift/origin/pull/7168/files#diff-4c376c2657fcf9dd607c5ca2fcae498b

move the import to match the changes in that pull